Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV PATH="$PATH_PYTHON3" VIRTUAL_ENV="$VIRTUAL_ENV_PYTHON3"
 
 # Install dependencies.
 COPY --chown=kat:kat requirements.txt /tmp/install/requirements.txt
-RUN install-requirements.py -d ~/docker-base/base-requirements.txt -d ~/docker-base/gpu-requirements.txt -r /tmp/install/requirements.txt
+RUN install_pinned.py -r /tmp/install/requirements.txt
```

Comment:
So previously the `base-requirements.txt` came from the Docker container itself, and now it's from the network / GitHub? Any implications of this change?

Reply:
That's a good point: it does slightly reduce reproducibility. On the other hand, when we update the base Docker container we rebuild the downstream images, so that should keep things in sync.

Once we've updated all the repos it might make sense to eliminate `base-requirements.txt` from the Docker image so that there aren't multiple sources for it.